charts/coturn/README.md

```diff
@@ -16,3 +17,12 @@ therefore recommended to run coturn on a separate Kubernetes cluster
 from the rest of the Wire services. Further details may be found in
 Wire's documentation for Restund, another TURN implementation, on
 [this](https://docs.wire.com/understand/restund.html#network) page.
+
+coturn can optionally be configured to expose a TLS control
+port. The TLS private key and certificate to use for this port can
+be supplied manually in `secrets.tls.cert` and `secrets.tls.key`, or
+[cert-manager](https://cert-manager.io) can be used to provision certificates
+automatically. If cert-manager is in use, and no Issuer or ClusterIssuer is
+provided in the cluster already, then this chart can create an issuer using
+Let's Encrypt, though the configuration of ACME challenge solvers must be
+provided by the operator manually.
```

Comment:
IMHO, this should be out of scope for our coturn chart.

Deploying and configuring cert-manager according to the environment is something that the cluster operator should do - we shouldn't create `Issuer` in the helm chart. Can you drop the end of the paragraph, and drop the creation of the `Issuer` resource?

Reply:
Fixed in [3cae51b](https://github.com/wireapp/wire-server/pull/2620/commits/3cae51b9080c3c316e57fa93fe396418c348f4c9)